analysis/tc_tackle_testapp_public_binary.go

```diff
@@ -45,58 +31,23 @@ var TackleTestappPublicBinary = TC{
 				Rule:        "local-storage-00001",
 				Incidents: []api.Incident{
 					{
-						File:    "/shared/bin/maven/java-project/src/main/java/io/konveyor/demo/config/ApplicationConfiguration.java",
-						Line:    8,
-						Message: "An application running inside a container could lose access to a file in local storage.",
-					},
-					{
-						File:    "/shared/bin/maven/java-project/src/main/java/io/konveyor/demo/ordermanagement/config/PersistenceConfig.java",
-						Line:    39,
-						Message: "An application running inside a container could lose access to a file in local storage.",
-					},
-					{
-						File:    "/shared/bin/maven/java-project/src/main/java/io/konveyor/demo/ordermanagement/config/PersistenceConfig.java",
-						Line:    40,
-						Message: "An application running inside a container could lose access to a file in local storage.",
-					},
-					{
-						File:    "/shared/bin/maven/java-project/src/main/java/io/konveyor/demo/ordermanagement/config/PersistenceConfig.java",
-						Line:    41,
-						Message: "An application running inside a container could lose access to a file in local storage.",
-					},
-					{
-						File:    "/shared/bin/maven/java-project/src/main/java/io/konveyor/demo/ordermanagement/config/PersistenceConfig.java",
-						Line:    42,
-						Message: "An application running inside a container could lose access to a file in local storage.",
-					},
-					{
-						File:    "/shared/bin/maven/java-project/src/main/java/io/konveyor/demo/ordermanagement/config/PersistenceConfig.java",
-						Line:    61,
-						Message: "An application running inside a container could lose access to a file in local storage.",
-					},
-					{
-						File:    "/shared/bin/maven/java-project/src/main/java/io/konveyor/demo/ordermanagement/config/PersistenceConfig.java",
-						Line:    62,
-						Message: "An application running inside a container could lose access to a file in local storage.",
-					},
-					{
-						File:    "/shared/bin/maven/java-project/src/main/java/io/konveyor/demo/ordermanagement/exception/handler/ExceptionHandlingController.java",
-						Line:    20,
+						File:    "/shared/bin/java-project/src/main/java/io/konveyor/demo/config/ApplicationConfiguration.java",
+						Line:    14,
```

Comment:
I have no idea why [this rule](https://github.com/konveyor/rulesets/blob/main/default/generated/cloud-readiness/70-local-storage.windup.yaml#L34) is being triggered [here](https://github.com/konveyor/tackle-testapp-public/blob/main/configuration-utils/src/main/java/io/konveyor/demo/config/ApplicationConfiguration.java#L14).